src/Sylius/Bundle/ResourceBundle/Controller/RequestConfigurationFactory.php

```diff
@@ -86,4 +87,13 @@ private function parseApiParameters(Request $request)
 
         return array_merge($request->attributes->get('_sylius', []), $parameters);
     }
+
+    /**
+     * @return array
+     */
+    public function getDefaultParameters()
```

Comment:
This is just for the phpspec test.
Since it's my first one, I wanted to be sure that $this->getDefaultParameters()->shouldHaveKey('sortable')  :)